### ISSUE TITLE
[Bug] Permission package 

### ISSUE THREAD

**Issue by siberfx** [Laravel-Backpack/CRUD#3603]
# Bug report

### What I did

I've seeded permissions and role groups, all configurations is set proper, it hides menu elements.

### What I expected to happen

it should hide all menu which is hidden by permission or the role (  backpack_user->hasPermissionTo('project_access') )

### What happened

but when it comes to hide sidebar_content elements on dashboard page it hides them, but when it comes to get into a menu item, some hidden items are becoming visible on sidebar.

### What I've already tried to fix it

I've checked auth.php, and also base.php, I ensure its not caused by a code override.

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
### LARAVEL VERSION:
v8.31.0@2aa5c2488d25178ebc097052c7897a0e463ddc35

### BACKPACK VERSION:
4.1.38@90cf9d0613848144c1e5f3822892bf2a7afa860f

**Comment by siberfx** [comment]
I have found the reason, but it requires your help :) 

when we create custom crud operations this bug occures.

**Comment by pxpm** [comment]
Hello @siberfx 

I am sorry but I could not reproduce your issue. 

What I did: 

Copied `ShowOperation` into `CustomShowOperation` and added `use CustomShowOperation` in the controller. 

Added to sidebar: `backpack_user()->hasPermissionTo('manage news')` (User does not have permission for this).

Go into my `CustomShowOperation`, item still hidden in sidebar. 

Might be a cache issue ? Try running `php artisan cache:clear`, and forcely clear permission cache when updating permissions via WebUI.

Let me know if I missed some step to reproduce it. 

Thanks,
Pedro
